website/content/en/docs/ansible/dev/webhooks.md

```diff
@@ -0,0 +1,166 @@
+# Adding Admission Webhooks to an Ansible-based Operator
+
+## Background
+
+For general background on what admission webhooks are, why to use them, and how to build them,
+please refer to the official Kubernetes documentation:
+https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/
+
+This guide will assume that you understand the above content, and that you have an existing admission
+webhook server. You will likely need to make a few modifications to the webhook server container.
+
+When integrating an admission webhook server into your Ansible-based Operator, we recommend that you
+deploy it as a sidecar container alongside your operator. This allows you to make use of the proxy
+server that the operator deploys, as well as the cache that backs it.
```

Comment:
WDYT?
```suggestion
server that the operator deploys, as well as the cache that backs it. The sidecar is defined in the `deploy/operator.yaml` such as:

```yaml
...
# This deploys the webhook
        - name: webhook
          # Replace this with the built image name
          image: "REPLACE_WEBHOOK_IMAGE"
          imagePullPolicy: "Always"
          volumeMounts:
          - mountPath: /etc/tls/
            name: webhook-cert
...            
```

```